src/pages/documentation/mobile-core/signal/tabs/index.md

```diff
@@ -5,13 +5,13 @@
 Add the [Mobile Core](../index.md) extension to your project using the app's Gradle file.
 
 ```java
-implementation 'com.adobe.marketing.mobile:sdk-core:1.+'
+implementation 'com.adobe.marketing.mobile:sdk-core:2.+'
```

Comment:
Should we include a warning here as we are in all the other extensions
```
Using dynamic dependency versions is not recommended for production apps. Refer to this [page](<path-to-gradle-dependencies-doc-under-resources>) for managing gradle dependencies.
```

Reply:
Sure. I have added it as an alert right under the heading.